server/controllers/profile.controller.js

```diff
@@ -131,6 +131,77 @@ class ProfileController {
     };
     return Response(res, 200, 'Successfully unfollowed user', user);
   }
+
+  /**
+   * get followers
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+
+  static async getFollowers(req, res) {
+    const {
+      user: { id: userId }
+    } = req;
+
+    const user = await User.findByPk(userId);
+
+    const followers = await user.getFollowers({
+      attributes: ['id', 'email', 'firstname', 'lastname'],
+    });
+
+    if (followers.length === 0) {
+      return Response(res, 200, 'User currently does not have followers');
+    }
+
+    const follower = {
+      count: followers.length,
+      followee: user.username,
+      followers: followers.map(follow => ({
```

Comment:
Well done @samorahno. I don't think we need to loop through the `followers` array any longer. Why not just return the `followers` array retuned by the magic function here directly since you have already specified the required attributes there.

Reply:
@cupidcoder, your suggestion is valid and will be implemented. I was trying to exclude the Follower column initially.